File: Day 3/0 Trim Sentence/solution.js
function trimSentence(sentence) {
  let start = 0;
  let end = sentence.length - 1;

  // Trim whitespace from the start of the sentence
  while (start < end && /\s/.test(sentence[start])) {
      start++;
  }

  // Trim whitespace from the end of the sentence
  while (end > start && /\s/.test(sentence[end])) {
      end--;
  }

  // Return the trimmed sentence
  return sentence.substring(start, end + 1);
}

File: Day 3/0 Trim Sentence/solution.cs
public string TrimSentence(string sentence)
{
    int start = 0;
    int end = sentence.Length - 1;
    
    // Trim whitespace from the start of the sentence
    //TODO: Bonus points: trim any char which is not an alphabet or digit
    while (start < end && Char.IsWhiteSpace(sentence[start]))
    {
        start++;
    }
    
    // Trim whitespace from the end of the sentence
    while (end > start && Char.IsWhiteSpace(sentence[end]))
    {
        end--;
    }
    
    //TODO: replace this substring method with your own method
    // Return the trimmed sentence
    return sentence.Substring(start, end - start + 1);
}

// write code to find median in array [1,2,3,4,5,6,7]
int[] arr = new int[1,2,3,4,5];
// find first index




